ui/src/app/views/notebook-redirect/component.ts

```diff
@@ -146,12 +228,14 @@ export class NotebookRedirectComponent implements OnInit, OnDestroy {
     return this.localizeNotebooks([]).flatMap((localDir) => {
       // Use the Jupyter Server API directly to create a new notebook. This
       // API handles notebook name collisions and matches the behavior of
-      // clicking "new notebook" in the Jupyter UI.
+      // clicking 'new notebook' in the Jupyter UI.
       const workspaceDir = localDir.replace(/^workspaces\//, '');
-      return this.jupyterService.postContents(
+      return this.jupyterService.putContents(
```

Comment:
What happens if you choose an existing notebook name here?

Reply:
I don't know, but we should probably make that give an error anyway :) 
Will add an error message there.